From 601dd02a90aacf7ebe6fdfb15b0d004cf6ad6c3b Mon Sep 17 00:00:00 2001
From: Binh Nguyen <binh.nguyen.jz@renesas.com>
Date: Tue, 17 Nov 2020 10:49:08 +0700
Subject: [PATCH 001/135] dt-bindings: arm: renesas: Document for RZ/G2L SoC

Add device tree bindings documentation for the Renesas RZ/G2L
RZ/G2L have two versions: r9a07g044l1(Single A55 core), r9a07g044l2
(dual A55 cores)

Signed-off-by: Binh Nguyen <binh.nguyen.jz@renesas.com>
---
 Documentation/devicetree/bindings/arm/shmobile.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/arm/shmobile.txt b/Documentation/devicetree/bindings/arm/shmobile.txt
index 7b83c02..39bd3b2 100644
--- a/Documentation/devicetree/bindings/arm/shmobile.txt
+++ b/Documentation/devicetree/bindings/arm/shmobile.txt
@@ -29,6 +29,10 @@ SoCs:
     compatible = "renesas,r8a774b1"
   - RZ/G2E (R8A774C0)
     compatible = "renesas,r8a774c0"
+  - RZ/G2L single A55 core (R9A07G044L1)
+    compatible = "renesas,r9a07g044l1"
+  - RZ/G2L dual A55 cores (R9A07G044L2)
+    compatible = "renesas,r9a07g044l2"
   - R-Car M1A (R8A77781)
     compatible = "renesas,r8a7778"
   - R-Car H1 (R8A77790)
-- 
2.7.4

